src/main/java/com/seailz/javadiscordwrapper/utils/URLS.java

```diff
@@ -24,6 +24,11 @@ public static class INTERACTIONS {
             public static final String CALLBACK = "/interactions/{interaction.id}/{interaction.token}/callback";
         }
 
+        public static class COMMANDS {
+            public static final String REGISTER_GLOBAL_COMMAND = "/applications/{application.id}/commands";
```

Comment:
add documentation
example:

```
/**
* Registers a global command
* @param id The ID of the application to register it for
*/

Reply:
is this the right file you meant to write this under?

Reply:
/**
* doc
*/
public static String foo

Reply:
will do